dynamicprefixvici.c

```diff
@@ -204,25 +134,21 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
     int opt;
     opterr = 0; // no error message by getopt
     arguments->pool_name = NULL;
-    arguments->new_prefix_env = NULL;
-    arguments->old_prefix_env = NULL;
+    arguments->new_prefix = NULL;
 
-    while((opt = getopt(argc, argv, ":p:n:o:h")) != -1)
+    while((opt = getopt(argc, argv, ":p:n:h")) != -1)
     {
         switch (opt)
         {
-        case 'h':// Help
+        case 'h': // Help
             Usage();
             exit(1);
             break;
         case 'p': // Pool name
             arguments->pool_name = optarg;
```

Comment:
After thinking about it one more time, we should probably switch -n and -p, such that -p is the prefix, and -n is the pool **n**ame